molecule/default/tests/test_openstack_services.py

```diff
@@ -2,75 +2,115 @@
-                        'nova']
-
-    # Getting the list of all services
-    # (returning the list of Munch Python dictionaries)
-    all_services_list = os_api_conn.list_services()
-
-    for service in checked_services:
-        filtered_services = list(filter(lambda d:
-                                        d['name'].lower() == service,
-                                        all_services_list))
-
-        if len(filtered_services) == 0:
-            services_not_found.append(service)
-        else:
-            if not filtered_services[0]['enabled']:
-                services_not_enabled.append(service)
-
-    assert len(services_not_found) < 1,\
-        "\nServices not found:\n{}\nServices not enabled\n{}\n" \
-        "All services returned by list_service api: \n" \
-        "{}".format(services_not_found,
-                    services_not_enabled,
-                    pformat(unmunchify(all_services_list)))
-
-    assert len(services_not_enabled) < 1, \
-        "\nServices not enabled\n{}\n" \
-        "All services returned by list_service api: \n" \
-        "{}".format(services_not_enabled,
-                    pformat(unmunchify(all_services_list)))
+
+    assert service_enabled('nova', host)
+
+
+@pytest.mark.test_id('d8e0c200-2a78-11e9-aad3-9cdc71d6c120')
+@pytest.mark.jira('asc-1505', 'asc-1555')
+def test_openstack_glance_service(host):
+    """Test to verify that Nova service is enabled
+
+    Args:
+        host (testinfra.host.Host): Testinfra host fixture
+    """
+
+    assert service_enabled('glance', host)
+
+
+@pytest.mark.test_id('c66d7b5e-2a78-11e9-aad3-9cdc71d6c120')
+@pytest.mark.jira('asc-1505', 'asc-1555')
+def test_openstack_neutron_service(host):
+    """Test to verify that Nova service is enabled
+
+    Args:
+        host (testinfra.host.Host): Testinfra host fixture
+    """
+
+    assert service_enabled('neutron', host)
+
+
+@pytest.mark.test_id('b82ef310-2a78-11e9-aad3-9cdc71d6c120')
+@pytest.mark.jira('asc-1505', 'asc-1555')
+def test_openstack_keystone_service(host):
+    """Test to verify that Nova service is enabled
+
+    Args:
+        host (testinfra.host.Host): Testinfra host fixture
+    """
+
+    assert service_enabled('keystone', host)
+
+
+@pytest.mark.test_id('aadcb274-2a78-11e9-aad3-9cdc71d6c120')
+@pytest.mark.jira('asc-1505', 'asc-1555')
+def test_openstack_swift_service(host):
+    """Test to verify that Nova service is enabled
+
+    Args:
+        host (testinfra.host.Host): Testinfra host fixture
+    """
+
+    assert service_enabled('swift', host)
+
+
+@pytest.mark.test_id('89d09a96-2a78-11e9-aad3-9cdc71d6c120')
+@pytest.mark.jira('asc-1505', 'asc-1555')
+def test_openstack_heat_service(host):
+    """Test to verify that Nova service is enabled
+
+    Args:
+        host (testinfra.host.Host): Testinfra host fixture
+    """
+
+    assert service_enabled('heat', host)
+
+# TODO: will add cinder service in the list after resolving the
+# TODO: os_version_major for healthcheck
```

Comment:
The `TODO` comments should reference tech debt tickets.

Reply:
Filed tech-debt ticket ASC-1627